apps/ejabberd/test/auth_jwt_SUITE.erl

```diff
@@ -43,25 +50,38 @@ suite() ->
 %%--------------------------------------------------------------------
 
 init_per_suite(Config) ->
-    ok = stringprep:start(),
-    ok = ejabberd_auth_jwt:start(?DOMAIN1),
     case list_to_integer(erlang:system_info(otp_release)) of
         Release when Release < 18 ->
             {skip, "jwerl need recent crypto features"};
         _ ->
-            Config
+            application:ensure_all_started(stringprep),
+            Self = self(),
+            ETSProcess = spawn(fun() -> ets_owner(Self) end),
+            wait_for_ets(),
+            meck_config(Config),
+            [{ets_process, ETSProcess} | Config]
     end.
 
 end_per_suite(Config) ->
-    ok = ejabberd_auth_jwt:stop(?DOMAIN1),
+    meck_cleanup(),
+    stop_ets(proplists:get_value(ets_process, Config)),
     Config.
 
+init_per_group(public_key, Config) ->
+    Root = filename:join(["..", "..", "..", ".."]),
```

Comment:
Maybe we should have `ct_helper:get_root_directory()` for this.
I just don't like to have such strange logic inside regular suites.